package.json

```diff
@@ -26,7 +26,7 @@
     "@lint-todo/utils": "^12.0.0",
     "chalk": "^4.1.0",
     "ci-info": "^3.3.0",
-    "eslint": "^7.10.0",
+    "eslint": "^8.21.0",
```

Comment:
The version that begins support for async formatters is [`8.4.0`](https://github.com/eslint/eslint/releases/tag/v8.4.0). We should ideally update to that, and additionally add a `peerDependency` that specifies this as the minimum version, to ensure consumers are correctly supported.